test/dotnet-build.Tests/GivenDotnetBuildBuildsProjects.cs

```diff
@@ -42,5 +42,19 @@ public void It_builds_projects_with_Unicode_in_path_project_path_passed()
                 .Should()
                 .Pass();
         }
+
+        [Fact]
+        public void It_builds_projects_with_spaces_in_the_path()
```

Comment:
End To End also tests with a space in the path, but I think a more explicit test is a good thing. 

This is a specific scenario which is failing (xmlDoc) so what do you think of naming the test to reflect that?